src/frontend/hooks/persistedState/usePersistedSettings.ts

```diff
@@ -5,21 +5,25 @@ import {CoordinateFormat} from '../../sharedTypes';
 type SettingsSlice = {
   coordinateFormat: CoordinateFormat;
   manualCoordinateEntryFormat: CoordinateFormat;
+  syncSetting: 'previews' | 'everything';
```

Comment:
suggestion: maybe define a shared type for this to reduce some keystrokes when working with this elsewhere?

Reply:
Ok. Done.